src/components/sections/Catalogue/CurrentProductSection/components/Articles/ProductDescription/components/ButtonsBlock.tsx

```
@@ -0,0 +1,48 @@
+          id='icon-cart'
+          w={18}
+          h={18}
+        />
+        <span className='font-manrope text-base font-semibold capitalize tracking-wide'>
+          Add to Cart
+        </span>
+      </SquareButton>
+
+      <SquareButton
+        className='flex w-60 justify-center gap-2 rounded-sm border-2 border-solid
+          border-brown-accent bg-white py-4 hover:bg-brown-accent'
```

ESLint

[error] Classname 'border-brown-accent' is not a Tailwind CSS class!

(tailwindcss/no-custom-classname)